pkgs/development/tools/poetry2nix/poetry2nix/overrides/default.nix

```diff
@@ -585,7 +585,7 @@ lib.composeManyExtensions [
 
       duckdb = super.duckdb.overridePythonAttrs (old: {
         postPatch = lib.optionalString (!(old.src.isWheel or false)) ''
-          cd tools/pythonpkg
+          ${lib.optionalString (lib.versionOlder old.version "0.8") "cd tools/pythonpkg"}
```

Comment:
I am confused by this postPatch. harlequin uses duckdb 0.8.1 and that derivation should already have postPatch set, however, when i delete this block the postPatch attribute is empty, as is the patches attribute. i wonder how it is getting cleared from the super derivation.

Reply:
Well, yes, but the super derivation here is the poetry2nix version of the duckdb python lib. This change aligns the override with the one in the poetry2nix repo.

Reply:
Basically, this doesn't use the same duckdb python package we worked on together in the other PR(s).